control-plane/api/v1alpha1/jwtprovider_types.go

```diff
@@ -432,6 +441,180 @@ func (r *RemoteJWKS) validate(path *field.Path) field.ErrorList {
 	}
 
 	errs = append(errs, r.RetryPolicy.validate(path.Child("retryPolicy"))...)
+	errs = append(errs, r.JWKSCluster.validate(path.Child("jwksCluster"))...)
+	return errs
+}
+
+type JWKSCluster struct {
+	// DiscoveryType refers to the service discovery type to use for resolving the cluster.
+	//
+	// This defaults to STRICT_DNS.
+	// Other options include STATIC, LOGICAL_DNS, EDS or ORIGINAL_DST.
+	DiscoveryType ClusterDiscoveryType `json:",omitempty" alias:"discovery_type"`
+
+	// TLSCertificates refers to the data containing certificate authority certificates to use
+	// in verifying a presented peer certificate.
+	// If not specified and a peer certificate is presented it will not be verified.
+	//
+	// Must be either CaCertificateProviderInstance or TrustedCA.
+	TLSCertificates *JWKSTLSCertificate `json:",omitempty" alias:"tls_certificates"`
+
+	// The timeout for new network connections to hosts in the cluster.
+	// If not set, a default value of 5s will be used.
+	ConnectTimeout time.Duration `json:",omitempty" alias:"connect_timeout"`
```

Comment:
I think the json tags here should have the camelCase alias and not the `alias` tags